lighthouse-extension/app/popup.html

```diff
@@ -38,15 +38,36 @@ <h2 class="header-titles__url">...</h2>
 
   <main class="main" role="main">
     <div class="feedback"></div>
-    <button class="generate-report">Generate report</button>
+
+    <button class="button button--link button--configure" id="configure-options">Options</button>
+    <button class="button button--generate" id="generate-report">Generate report</button>
   </main>
 
-  <aside class="status">
+  <aside class="status subpage">
     <div class="status__spinner"></div>
     <div class="status__msg">Starting...</div>
     <div><small class="status__detailsmsg"></small></div>
   </aside>
 
+  <aside class="options subpage">
+    <h2 class="options__title">Options</h2>
+    <ul class="options__list">
+        <li><input type="checkbox" id="offline" checked="checked" value="App can load on offline/flaky connections" /><label for="offline">App can load on offline/flaky connections</label></li>
+        <li><input type="checkbox" id="pageload" checked="checked" value="Page load performance is fast" /><label for="pageload">Page load performance</label></li>
+        <li><input type="checkbox" id="progressive" checked="checked"  value="Site is progressively enhanced" /><label for="progressive">Site is progressively enhanced</label></li>
+        <li><input type="checkbox" id="secure" checked="checked"  value="Network connection is secure" /><label for="secure">Network connection is secure</label></li>
+        <li><input type="checkbox" id="homescreen" checked="checked" value="User can be prompted to Add to Homescreen" /><label for="homescreen">User can be prompted to Add to Homescreen</label></li>
```

Comment:
its slightly easier to wrap the checkbox with the `<label>` to avoid the [for] and [id] attributes.